web/client/product/plugins/Attribution.jsx

```diff
@@ -1,44 +1,104 @@
-const PropTypes = require('prop-types');
-/**
+/*
  * Copyright 2016, GeoSolutions Sas.
  * All rights reserved.
  *
  * This source code is licensed under the BSD-style license found in the
  * LICENSE file in the root directory of this source tree.
  */
 const React = require('react');
-const src = require("./attribution/geosolutions-brand.png");
+const PropTypes = require('prop-types');
+const src = require("./attribution/geosolutions-brand-sm.png");
 const assign = require('object-assign');
+const tooltip = require('../../components/misc/enhancers/tooltip');
+const {isString, trimStart} = require('lodash');
+const {Nav, NavItem, Glyphicon} = require('react-bootstrap');
+const Img = tooltip(props => (<img {...props}/>));
+const ContainerDimensions = require('react-container-dimensions').default;
+const NavItemT = tooltip(NavItem);
 
 class Attribution extends React.Component {
```

Comment:
This plugin is both NavMenu and Attribution logo. We should separate the 2 concepts to render attribution (logo and so on) in NavMenu, and not vice-versa.
We want to use it in dependent projects that do not have the logo on top.